HOWTORELEASE.md

```diff
@@ -0,0 +1,14 @@
+# Release Procedure
+
+These are the steps to make a release for version `<version_number>`:
+
+1. Create a commit with title `Bump to v<version_number>` with the following changes:
+    1. Update `CHANGELOG.md` with all important changes introduced since previous version.
```

Comment:
in outlines, it's considered suboptimal to have `1.` and `2.` numbering under `1.`. Can we change the sublevel numbering to `a.`, `b.` ?

Reply:
The modifications of file `CHANGELOG.md` have to adhere to some kind of syntax, right? Would it make sense to mention that and add a link to the changelog syntax definition?

Reply:
yes we can!

Reply:
done

Reply:
The changelog file has the link to syntax and structure